### DISCUSSION TITLE
Update from the old Blender Launcher?

### DISCUSSION THREAD

**Discussion by Xiarno** [Victor-IX/Blender-Launcher-V2#84] [Q&A] [answered]
I have the old Blender Launcher, the one that got archived. How to update to this new version as seamlessly as possible?

**Comment by Victor-IX** [comment, marked as answer]
It's very easy to do:

1. Download the latest release for your OS from the [releases page](https://github.com/Victor-IX/Blender-Launcher-V2/releases/latest).
2. Quit any running instance of Blender Launcher.
3. Unpack the Blender Launcher.exe file and replace the existing one.
4. Launch the new version of the Blender Launcher.
5. Set back your [library folder](https://victor-ix.github.io/Blender-Launcher-V2/library_folder/).

For more information, you can check the [documentation](https://victor-ix.github.io/Blender-Launcher-V2/).

**Reply by Xiarno** [reply]
Thanks!
Is this normal?
![14134_Blender_Launcher_G9WEKJjOBn](https://github.com/Victor-IX/Blender-Launcher-V2/assets/24660485/a3e75431-e81b-4bab-9d58-b21955f6efce)

**Reply by Victor-IX** [reply]
Yes, Blender is currently under a heavy load and blocks the gathering of the Stable build. 
So stable builds are currently not available... Hopefully, the issue will be resolved soon in the meantime you can download stable builds manually on the website and add them to the Stable folder of the Blender launcher

**Reply by Xiarno** [reply]
I see. Thanks for the help!

**Reply by Xiarno** [reply]
> Yes, Blender is currently under a heavy load and blocks the gathering of the Stable build. So stable builds are currently not available... Hopefully, the issue will be resolved soon in the meantime you can download stable builds manually on the website and add them to the Stable folder of the Blender launcher

Is there any news about it?

**Reply by Victor-IX** [reply]
No... I got no response on their discord, I'll try to reach out to them through another way on Monday to find out what is currently happening

**Reply by Xiarno** [reply]
Thanks!
